avro/swagger/pom.xml

```diff
@@ -59,6 +49,11 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.cerner.beadledom</groupId>
+      <artifactId>beadledom-swagger2</artifactId>
```

Comment:
Instead of having a dependency on beadledom-swagger2, lets update to have a dependency on swagger, 

```
    <dependency>
      <groupId>io.swagger</groupId>
      <artifactId>swagger-core</artifactId>
    </dependency>
    <dependency>
      <groupId>io.swagger</groupId>
      <artifactId>swagger-models</artifactId>
    </dependency>

    ...

    <dependency>
      <groupId>io.swagger</groupId>
      <artifactId>swagger-jaxrs</artifactId>
      <scope>test</scope>
    </dependency>
```

Reply:
269adfc3